app/src/main.c

```diff
@@ -1009,6 +1009,8 @@ void main(void)
 	}
 #endif
 
+	init_timers(ctx);
```

Comment:
Good catch ! Could you we move it even earlier, for example between load_config() and init_accel(), around line 950  ?